CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/.git)
   if(${GIT_VERSION_RESULT})
     message(WARNING "Error running git describe to determine version")
   else()
-    set(CMAKE_PROJECT_VERSION "${CMAKE_PROJECT_VERSION} (${GIT_VERSION})")
```

Comment:
I think this we will want to replace all CMAKE_PROJECT_VERSION in other places too since this code only runs if you have a git checkout and `git describe` works.

if PROJECT_VERSION works in all cases I would just do a global search and replace.

Reply:
Sounds good, have done in the updated commit. Definitely works from git checkout, and should now also work on non-git-checkout path as well.